multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/ScaleAppStep.java

```diff
@@ -27,7 +28,12 @@ protected StepPhase executeStep(ProcessContext context) {
         CloudControllerClient client = context.getControllerClient();
 
         String appName = app.getName();
-        Integer instances = (app.getInstances() != 0) ? app.getInstances() : null;
+
+        if (app.getInstances() == 0) {
```

Comment:
Did you try to deploy mta withhout specified instances parameter? What will happen if instances parameter is negative value?

Reply:
The deployment of mta with no specified instances parameter is tested and uses the default instance count. In the case of the instances parameter being a negative value, a SLException is thrown.